lib/rpc.ex

```diff
@@ -163,4 +167,22 @@ defmodule Tezex.Rpc do
       {:error, _} = err -> err
     end
   end
+
+  def post(%Rpc{} = rpc, path, body) do
+    url =
+      URI.parse(rpc.endpoint)
+      # 🚧 Try to reproduce the same API as the rest of this file but it doesn't make sense for the injection endpoint since it doesn't include the /chains/id
```

Comment:
right, it differs from GET endpoints, here the chain should be passed as query param: https://gitlab.com/tezos/tezos/-/blob/master/docs/api/rpc-openapi-rc.json#L1778-1785

Reply:
This parameter is weird, I thought it can select which chain you want to query but it's not like that. For example if you try to query `https://ghostnet.tezos.marigold.dev/` with a `chain=test` it won't work:

```elixir
rpc = %Tezex.Rpc{endpoint: "https://ghostnet.tezos.marigold.dev/", chain_id: "test"}
Tezex.Rpc.get_next_counter_for_account(rpc, "tz1ZW1ZSN4ruXYc3nCon8EaTXp1t3tKWb9Ew")
```

RPC will return a 200 with an empty body. I think it make sense since here we want to query the main chain of this ghostnet endpoint, not the test one :shrug: (or at least that what I understand).

Anyway I removed the comment and updated the implementation :sparkles: